File: video/users/client-initialization/client-initialization.js
// Create an AccessManager to manage our Access Token
const accessManager = new Twilio.AccessManager('$TWILIO_ACCESS_TOKEN');

// Create a Conversations Client and connect to Twilio's backend
conversationsClient = new Twilio.Conversations.Client(accessManager);
conversationsClient.listen().then(
  function() {
    console.log('Connected to Twilio!');
  },
  function(error) {
    console.log('Could not connect to Twilio: ' + error.message);
  }
);

File: video/users/client-initialization/client-initialization.java
// Create an AccessManager to manage our Access Token
AccessManager accessManager = new AccessManager(ConversationActivity.this,
                                ACCESS_TOKEN,
                                accessManagerListener());

// Create a Conversations Client and connect to Twilio's backend.
TwilioConversationsClient conversationsClient =
  TwilioConversationsClient.create(accessManager, conversationsClientListener());
conversationsClient.listen();

/* See the "Working with Conversations" guide for instructions on constructing a
ConversationsClientListener */
private TwilioConversationsClient.Listener conversationsClientListener() {
  return new TwilioConversationsClient.Listener() {
    @Override
    public void onStartListeningForInvites(TwilioConversationsClient conversationsClient) {
      Log.i(TAG, "Connected to Twilio!");
    }

    ...

  };
}
